app.py

```diff
@@ -8,5 +11,81 @@ def hello_world():
     return 'Hello World!'
 
 
+@app.route('/register', methods=['POST'])
+def create_user():
+    """
+    Should create user
```

Comment:
Should be a declaration of what method does, not a posibility

Reply:
It's not actually creating a user right now, it should create one phase 2